src/components/data/PathListAutomation.js

```diff
@@ -193,246 +199,276 @@ export default function PathListAutomation(props) {
+                                                        )}
+                                                </List>
+                                            </Paper>
+                                        </Grid>
+                                        <Grid item xs>
+                                            <Paper
+                                                style={{
+                                                    padding: theme.spacing(1),
+                                                }}
+                                            >
+                                                <Field
+                                                    startingPath={startingPath}
+                                                    name="dest"
+                                                    id={buildId(
+                                                        baseId,
+                                                        ids.PATH_LIST_AUTO_DEST_FIELD
+                                                    )}
+                                                    acceptedType={
+                                                        ResourceTypes.FOLDER
+                                                    }
+                                                    label={t("pathListDestLbl")}
+                                                    component={InputSelector}
+                                                    required={true}
+                                                />
+                                                <Field
+                                                    id={buildId(
+                                                        baseId,
+
+                                                        ids.PATH_LIST_AUTO_FILE_NAME_FIELD
+                                                    )}
+                                                    name="fileName"
+                                                    required={true}
+                                                    label={t("fileName")}
+                                                    component={SaveAsField}
+                                                />
+                                            </Paper>
+                                        </Grid>
+                                    </Grid>
+                                </DialogContent>
+                                <DialogActions>
+                                    {createPathListError && (
+                                        <Grid item xs>
+                                            <ErrorTypographyWithDialog
+                                                baseId={baseId}
+                                                errorMessage={errorMsg}
+                                                errorObject={
+                                                    createPathListError
+                                                }
```

Comment:
🤔 I see why you added this here - that appears to be how it was originally added.  I suspect it was intended to go within `DialogContent` though and was missed during review. The idea being that if there's an error, you can see it at the top and bottom of the dialog content, so it's harder to miss.

Reply:
Putting it along the action buttons in a `Grid`, will align the error message with the action buttons. At least that what I remember.